test/unit/reducers/index.js

```diff
@@ -0,0 +1,9 @@
+import reduxInconsistentApi from '../../../src/redux-inconsistent-api';
```

Comment:
Follow-up to that comment: all I did was grab the npm package. Didn't update any of the code yet. #4 is about finishin the job.